source/pkg/reconciler/streamsource/controller.go

```diff
@@ -61,7 +91,7 @@ func NewController(
 		ssr:                 &reconciler.StatefulSetReconciler{KubeClientSet: kubeclient.Get(ctx)},
 		rbr:                 &reconciler.RoleBindingReconciler{KubeClientSet: kubeclient.Get(ctx)},
 		sar:                 &reconciler.ServiceAccountReconciler{KubeClientSet: kubeclient.Get(ctx)},
-		configs:             reconcilersource.WatchConfigurations(ctx, component, cmw),
+		configs:             reconcilersource.WatchConfigurations(ctx, component, cmw, WithRedis),
```

Comment:
@lionelvillard is this the recommended way of adding new CMs to controller watch?

Reply:
that looks good to me :-)